react-native-cli/index.js

```diff
@@ -198,7 +199,14 @@ function run(root, projectName, logLevel) {
   if (logLevel === 'debug' || logLevel === 'verbose') {
     spawnArgs = {stdio: 'inherit'};
   }
-  var proc = spawn('npm', args, spawnArgs);
+  var proc;
+  if (os.platform() === 'win32'){
+    args.unshift('npm');
+    args.unshift('/c');
+    proc = spawn('cmd', args, spawnArgs);
```

Comment:
To run things via shell, there is https://nodejs.org/dist/latest-v4.x/docs/api/child_process.html#child_process_child_process_exec_command_options_callback. Wouldn't it be safer to use that instead of hardcoding `cmd /c`?